src/qt/glcanvas.cpp

```diff
@@ -348,6 +350,30 @@ bool wxGLContext::SetCurrent(const wxGLCanvas&) const
     return false;
 }
 
+//---------------------------------------------------------------------------
+// PanGestureRecognizer - helper class for wxGLCanvas
+//---------------------------------------------------------------------------
+
+class PanGestureRecognizer : public QGestureRecognizer
+{
+private:
+   static const int MINIMUM_DISTANCE = 10;
+
+   typedef QGestureRecognizer parent;
+
+   bool IsValidMove(int dx, int dy);
+
+   virtual QGesture* create(QObject* pTarget);
+
+   virtual QGestureRecognizer::Result recognize(QGesture* pGesture, QObject *pWatched, QEvent *pEvent);
+
+   void reset (QGesture *pGesture);
+
+   uint m_started;
```

Comment:
This doesn't seem to be actually used anywhere (its use is commented out), could it please be removed completely?

It also looks like it ought to be a `bool` if it were used.

Reply:
Done.